doc/source/development/rfc/rfc108_driver_removal_3_11.rst

```diff
@@ -0,0 +1,160 @@
+
+Technical solution
+------------------
+
+Fully removed drivers:
+
+- raster:
+
+  * JP2Lura: JPEG2000-capable driver. Was based on a proprietary SDK.
+    Initial proponents of that driver no longer use it.
+  * OZI OZF2/OZFX3: OZI now uses a new format v4 that has not been
+    reverse engineered.
+  * BLX (Magellan BLX Topo File Format). No activity in this driver from
+    non-GDAL team. No obvious sign in issue tracker it is used.
+  * XPM driver: Not relevant in a geospatial context. Obsolete image format.
+    Users can probably use ImageMagick if support for it is needed.
+  * SGI driver: same as above
+  * FIT driver: No idea what it is about  (not to be confused by FITS, which
+    is actively used by astronomic/planetary community)
+  * DIPEx: No idea what it is about
+  * CTable2 driver: obsolete PROJ.4 grid format. There was just one file ("null")
+    known to use that format.
+  * R (R Object Data Store (``*.rda``)): not known to be used. Not to be confused with
+    the RRaster driver which is used in the R ecosystem.
+  * GSAG (Golden Software ASCII) and GSBG (Golden Software 6.0 binary): likely
+    obsolete. GS7B (Golden Software 7.0 binary) kept as there are signs it is
+    used.
+  * Rasterlite: Rasterlite 1.0 format. Obsolete/no longer maintained by the
+    Spatialite crew. The SQLite driver has support for Rasterlite 2
+  * BT (VTP .bt Binary Terrain Format). The VTP project seems abandoned since
+    2015.
+  * SDTS: obsolete
+  * RDB (RIEGL Laser Measurement Systems GmbH): Not CI tested. Depends on
+    proprietary SDK. Removed with agreement from RIEGL.
+
+- vector:
+
+  * Geoconcept export .txt: no sign it is used. Feedback from a French GIS
+    forum when asking if someone still uses the driver: "It is not clear who still
+    uses Geoconcept (the software) at all"
+  * OGDI: Vector Product Format / DGN / VMap. All obsolete formats. The
+    problematic part is that it depends on libogdi, which is an obsolete C
+    code base nobody sane would want to maintain anymore.
+  * SDTS: obsolete
+  * SVG: only supported files from Cloudmade Vector Stream Server, which no
+    longer exists. The code could probably be re-enabled to be made more generic
+    and handle arbitrary SVG, if the needs arises.
+  * Tiger: pre-2006 format for US Census. New datasets are in shapefile.
+
+Write-support removal:
+
+- raster:
+
+  * ADRG: defense-related format. Write support was mostly to test
+    the read side of the driver
+  * ELAS: not sure what it is about and that this is actively used
+  * PAux: not sure what it is about and that this is actively used
+  * MFF: not sure what it is about and that this is actively used
+  * MFF2/HKV: not sure what it is about and that this is actively used
+  * LAN: not sure what it is about and that this is actively used
+  * BYN: Canadian grid-related format. Write support doesn't seem useful
```

Comment:
LAN is an old ERDAS format before HFA (.img).  It is a simple format, probably "raw", and it does not support compression. Write support doesn't seem useful.

Reply:
I've integrated your comment. Thanks